cartography/intel/gcp/dns.py

```diff
@@ -75,7 +75,8 @@ def get_dns_rrs(dns: Resource, dns_zones: List[Dict], project_id: str) -> List[R
                 for resource_record_set in response['rrsets']:
                     resource_record_set['zone'] = zone['id']
                     resource_record_set[
-                        "id"] = f"projects/{project_id}/resourceRecordSet/{resource_record_set.get('name',None)}"
+                        "id"
```

Comment:
can we format this line a little bit?

Reply:
when linter test performed some part of this line gose to the nest line